packages/cli/src/utils/misc.ts

```diff
@@ -4,6 +4,7 @@ import { hasProperty } from '@metamask/utils';
 import rfdc from 'rfdc';
 import { Arguments } from 'yargs';
 
+// TODO: Use `utils` package.
 export const deepClone = rfdc({ proto: false, circles: false });
```

Comment:
Any reason not to do that now?

Reply:
Nope, I put it there as a reminder for myself, but forgot. 😅